apps/pie-docs/src/patterns/cookie-banner/overview/overview.md

```diff
@@ -0,0 +1,247 @@
+src:"../../../assets/img/patterns/cookie-banner/anatomy-modal.svg",
+alt: "Anatomy of the Modal.",
+width: 120
+} %}
+
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Banner:** Contains the content.",
+        "**Title:** Provides context to the Banner.",
+        "**Body copy:** Provides supporting context to the user.",
+        "**Call to actions:** Allow the user to proceed with multiple options.",
+        "**Modal:** A closeable window that overlays a page.",
+        "**Content:** Provides information for the user about the cookie preference options available.",
+        "**“Save” button:**  Allows the user to save their preferences and continue with their flow.",
+        "**Scrim:** Screen overlay that obscures the on-page content."
+    ]
+} %}
+
+---
+
+## Variations
+
+### Banner
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/variations-banner.svg",
+alt: "Cookie banner with copy about cookies and buttons to accept or manage preferences.",
+width: 120
+} %}
+
+### Modal
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/variations-modal.svg",
+alt: "Modal with options to authorise cookies.",
+width: 120
+} %}
+
+___
+
+## Modifiers
+
+### Banner
+
+#### Default
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/modifier-default.svg",
+alt: "Default cookie banner, with Accept All as a primary action only.",
+width: 120
+} %}
+
+#### Primary actions only
+
+This version is used when legal compliance in the country requires you to do so. In countries that do not require this, the default variation is used.
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/modifier-primary-actions.svg",
+alt: "Cookie banner, with Accept All and Necesary Only as a primary actions.",
+width: 120
+} %}
+
+---
+
+## Interaction area
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/interaction-area-banner.svg",
+alt: "Banner with highlights on the areas of interaction: links and buttons.",
+width: 120
+} %}
+
+{% contentPageImage {
+src:"../../../assets/img/patterns/cookie-banner/interaction-area-modal.svg",
+alt: "Modal with highlights on the areas of interaction: links and buttons.",
+width: 120
+} %}
+
+{% list {
+    type: listTypes.highlight,
+    highlightColour: ["support-brand-03", "support-brand-04", "support-brand-07", "support-brand-05", "support-brand-06"],
+    items: [
+        "**Redirect:**  It will redirect the user to the page  they have selected in a new tab, the cookie banner would appear until the user makes a selection.",
+        "**Confirm selection:**  It saves the user's selection and closes the cookie banner/modal.",
+        "**Opens a “Manage preferences” modal:**  A modal will open and allow user to manage their preferences. ",
+        "**Toggle on/off:**  Allows the user to toggle selections on or off.",
+        "**Back button:**  It closes the modal and resurfaces the cookie banner."
```

Comment:
A few suggestions:
```suggestion
        "**Redirect:** Redirects the user to the page they have selected in a new tab, the cookie banner will continue to show until the user makes a selection.",
        "**Confirm selection:** Saves the user's selection and closes the modal and cookie banner.",
        "**Opens a “Manage preferences” modal:** A modal will open to allow the user to manage their preferences.",
        "**Toggle on/off:** Allows the user to toggle selections on or off.",
        "**Back button:** Closes the modal and shows the cookie banner again."
```

Also, I think this is quite pretty on the page, but I'm not sure how accessible it is in terms of colourblindness?